crates/saya/core/src/lib.rs

```diff
@@ -241,13 +252,13 @@ impl From<starknet::providers::ProviderError> for error::Error {
 
 //     #[tokio::test]
 //     async fn test_herodotus_verify() {
-//         let proof = prove(EXAMPLE_STATE_DIFF.into(), ProverIdentifier::Stone).await.unwrap();
-//         let _tx = verify(proof, VerifierIdentifier::HerodotusStarknetSepolia).await.unwrap();
+//         let proof = prove(EXAMPLE_STATE_DIFF.into(), ProverIdentifier::Stone).await?;
+//         let _tx = verify(proof, VerifierIdentifier::HerodotusStarknetSepolia).await?;
 //     }
 
 //     #[tokio::test]
 //     async fn test_local_verify() {
-//         let proof = prove(EXAMPLE_STATE_DIFF.into(), ProverIdentifier::Stone).await.unwrap();
-//         let _res = verify(proof, VerifierIdentifier::StoneLocal).await.unwrap();
+//         let proof = prove(EXAMPLE_STATE_DIFF.into(), ProverIdentifier::Stone).await?;
+//         let _res = verify(proof, VerifierIdentifier::StoneLocal).await?;
```

Comment:
We may remove those tests for now, and we should use the http stone once available for testing. 👍

Reply:
fixed